rbac/common/role/sync_direction.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 Contributors to Hyperledger Sawtooth
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+"""Implements setting the sync_direction for roles"""
+import time
+import rethinkdb as r
+from rbac.common.logs import get_default_logger
+from rbac.providers.common.db_queries import connect_to_db
+
+LOGGER = get_default_logger(__name__)
+
+
+def set_sync_direction(role_id, direction):
+    """Sets the sync_direction of role_id.
+
+    Args:
+        role_id: The next_id of the role.
+        direction: str with value of "INBOUND" or "OUTBOUND"
+
+    Returns:
+        RethinkDB output of the update query.
+    """
+    conn = connect_to_db()
+    for _ in range(3):
```

Comment:
This goes a bit against the established pattern using while loops.

Reply:
Fair enough, I'll switch this and the other to while loops.